src/main/java/seedu/recipe/model/Date.java

```diff
@@ -44,6 +44,10 @@ public Date(LocalDate date) {
         this.date = date;
     }
 
+    public Date() {
+        this.date = LocalDate.now();
```

Comment:
Might want to call today() instead, which would set the LocalDate to Singapore timezone and return a new Date object.